### DISCUSSION TITLE
Snapshot is failing to css hash keep changing

### DISCUSSION THREAD

**Discussion by antonioaltamura** [ant-design/ant-design#46386] [General]
Our CI/CD pipeline is failing because of hash changes `css-dev-only-do-not-override-xxxx`. I see there are some discussions around this already, but I can't find a solution.

      <ConfigProvider theme={{ hashed: false }}></ConfigProvider>

`hashed:false` didn't really solve in Antd 5.11. Hashes are still there?

**Comment by zombieJ** [comment]
`hashed: false` is the answer. If still meet the css hash, you may use static function which not consume the context.

**Reply by antonioaltamura** [reply]
Thanks for the answer. Would you expand a bit your suggestion?

**Comment by antonioaltamura** [comment]
Related: https://github.com/ant-design/ant-design/issues/38660

**Comment by antonioaltamura** [comment]
I'm still not clear with the suggested solution...
